src/resources/views/crud/fields/select2_from_ajax.blade.php

```diff
@@ -6,17 +6,16 @@
     // by default set ajax query delay to 500ms
     // this is the time we wait before send the query to the search endpoint, after the user as stopped typing.
     $field['delay'] = $field['delay'] ?? 500;
+    $field['allows_null'] = $field['allows_null'] ?? $crud->model::isColumnNullable($field['name']);
 @endphp
 
 @include('crud::fields.inc.wrapper_start')
     <label>{!! $field['label'] !!}</label>
-    <?php $entity_model = $crud->model; ?>
-
     <select
         name="{{ $field['name'] }}"
         style="width: 100%"
         data-init-function="bpFieldInitSelect2FromAjaxElement"
-        data-column-nullable="{{ $entity_model::isColumnNullable($field['name'])?'true':'false' }}"
+        data-column-nullable="{{ var_export($field['allows_null']) }}"
```

Comment:
I would rather do a ternary conditional here, I've had problems with misusing `var_export` in the past so I'm now wary of using it 😂 For example, shouldn't it be using the second parameter here, so that it returns the value not output it, since we're in a blade echoing block ```{{ }}```?

I'm not sure this will work as-is, please double-check guys.

My suggestion (also untested) would be something like this which I think is safer and reads easier:
```suggestion
        data-column-nullable="{{ $field['allows_null']?'true':'false' }}"
```

Reply:
@tabacitu atm we use `var_export` in 16 places in 6 files:

![image](https://user-images.githubusercontent.com/7188159/103760064-7fddbd00-500c-11eb-8148-7290eb40fd23.png)

I am creating a separate PR to remove all `var_exports` in favor of using ternary operators. I think it would be better if we do it once and for all, otherwise someplaces will have `var_export`, others ternarys, others who knows 👍 

AFAIK about `var_export` in this scenario they both behave the same, with or without second parameter. 

You can do a quick test like

```php
test="{{var_export($field['allows_null'])}}"
test2="{{var_export($field['allows_null'], true)}}"
    @dd('test end')
```
![image](https://user-images.githubusercontent.com/7188159/103761600-d3e9a100-500e-11eb-804d-778134ddce5f.png)
 ⏫ See the results at the end of the image? They both produce the same end result (I think here it's because of blade echoing)

But I do agree with the refactor, I think it's more readable using ternary operators.

Best,
Pedro

Reply:
@pxpm I created an issue so we could move forward https://github.com/Laravel-Backpack/CRUD/issues/3460.
Do you think we should fix this one? Or should we fix everything together?

Reply:
I vouche for fixing them all in #3460 

Refactoring here is not in scope of this PR, and like I said in previous comment, `var_export` in this scenarios is not broken, so nothing to fix here, just refactor. 

If you agree with me, let's leave this as is, and refactor all `var_exports` in a separate PR.

Reply:
Excellent! Totally agree with both of you.